modules/@ergonode/ui/src/components/Form/FormHeader.vue

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE for license details.
+ */
+<template>
+    <h2
+        class="form-header"
+        v-text="text" />
+</template>
+
+<script>
+export default {
+    name: 'FormHeader',
+    props: {
+        text: {
```

Comment:
we use everywhere titile or label names for strings, I would stick with same naming